lib/ava-files.js

```diff
@@ -257,6 +266,67 @@ function handlePaths(files, excludePatterns, globOptions) {
 		});
 }
 
+function handlePathsSync(files, excludePatterns, globOptions) {
```

Comment:
Async and sync versions of `handlePath` need to be split out into functions to DRY it up.